frontend/src/common/components/header/components/ShoppingCartButton.tsx

```diff
@@ -0,0 +1,28 @@
+import { faShoppingCart } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import Badge, { BadgeProps } from '@mui/material/Badge';
+import { IconButton, styled } from "@mui/material";
+
+const StyledBadge = styled(Badge)<BadgeProps>(() => ({
+  '& .MuiBadge-badge': {
```

Comment:
Use a scss stylesheet instead for ShoppingCartButton, rather than styled() paradigm

Reply:
Move the css into that stylesheet, and name the stylesheet "ShoppingCartButton.scss" and place in the same "components" folder location beside ShoppingCartButton.tsx

Reply:
Also refer to the UX hifi for where to place the count (should be beside the cart icon rather than inside a circled badge), might want to reconsider which HTML elements/components to use and where to place them

Reply:
done - everything updated according to your suggestions